code/extensions/che-remote/package.nls.json

```diff
@@ -3,6 +3,7 @@
 	"description": "Provides remoteIndicator commands",
 	"openDocumentationCommand": "Eclipse Che: Open Documentation",
 	"openDashboardCommand": "Eclipse Che: Open Dashboard",
+	"openClusterConsoleCommand": "Eclipse Che: Open Cluster Console",
```

Comment:
Can we rely that it's always `OpenShift` console?
or
it could have another value?

I'm asking because I see that [on the dashboard side there is specific env variable](https://github.com/eclipse-che/che-dashboard/blob/b229ff4598d2af3990a8daa0cd71bdf061ec740d/packages/dashboard-backend/src/routes/api/clusterInfo.ts#L28) which contains a title. The dashboard reads that env variable and provides the title based on the given value.

About the `che-code` side - we could use that env variable as well **if it's really required** (so, title can have different value). But then the more complex solution needed:
- VS Code doesn't allow dynamic titles for commands provided by an extension (the names are defined in the `package.json` file of the extension)
- so, some part should be provided on the VS Code side + some part on an extension side

Reply:
There are other Kubernetes GUIs but they are not named "Console" (c.f. [the Kubernetes Dashboard](https://kubernetes.io/docs/tasks/access-application-cluster/web-ui-dashboard/) so "Cluster Console" is not a generic enough name and something like "Kubernetes GUI" is generic enough but it may be overkill considered that we can retrieve the console only if it's OpenShift.

That said we should:
- hide the menu item / command if VS Code is **not** running on OpenShift 
- use the "OpenShift Console" label